.github/actions/enumerate-tests/action.yml

```diff
@@ -49,7 +49,7 @@ runs:
           "/t:Build;ExtractTestClassNames"
           /bl:${{ github.workspace }}/artifacts/log/Debug/BuildTemplatesTests.binlog
           -p:ExtractTestClassNamesForHelix=true
-          -p:ArchiveTests=true
+          -p:RunHelixTests=true
```

Comment:
Maybe `PrepareForHelixTests` because this will only prepare the archives, and not actually send anything to helix.

Reply:
Renamed